src/vs/workbench/contrib/notebook/browser/contrib/find/notebookFindReplaceWidget.ts

```diff
@@ -150,7 +151,7 @@ class NotebookFindFilterActionViewItem extends DropdownMenuActionViewItem {
 	}
 }
 
-class NotebookFindInput extends FindInput {
+export class NotebookFindInputFilter extends Disposable {
 	private _filterButtonContainer: HTMLElement;
```

Comment:
The names are getting a bit confusing now. It's hard to tell which is the State and which one is the View.

Reply:
I'm a little confused about what you mean by this. I believe that I moved any state-related fields that had to do with the filter to the `NotebookFindInputFilter`, so there should be both state and view fields in the `NotebookFindInputFilter `?

Reply:
Ah, were you talking about `NotebookFindInputFilter` vs `NotebookFindFilters`? I renamed `NotebookFindInputFilter` to  `NotebookFindInputFilterButton` to try to reduce confusion.